### PR TITLE
chore(auto-updates): show toast when app is updated COMPASS-7686

### PR DESCRIPTION
**Pull request by mabaasit**
In this PR we added a new toast which is shown when compass has been updated. In addition,  we added the version which is being downloaded to the toast notification. (the versions in screenshots are random)

<table>
<tr>
 <td> Before
 <td> After
<tr>
 <td> 
image

 <td> 
image

<tr>
 <td> No toast previously
 <td> 
image

</table>

### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass/src/app/components/update-toasts.tsx [hunk 1]
```diff
@@ -6,18 +6,14 @@ import {
...
 const containerStyles = css({
   display: 'flex',
   flexDirection: 'row',
-  div: {
-    display: 'flex',
-    flexDirection: 'column',
-    margin: 'auto',
-    padding: spacing[1],
-  },
+  gap: spacing[100],
```
Comment (mcasimir) [inline packages/compass/src/app/components/update-toasts.tsx:16]: spacing[100]?

Comment (mabaasit) [reply packages/compass/src/app/components/update-toasts.tsx:16]: the [new units](https://www.mongodb.design/component/tokens/example/), existing 1 (4px) is new 100

File: packages/compass/src/app/components/update-toasts.tsx [hunk 2]
```diff
@@ -61,37 +77,66 @@ const RestartCompassToastContent = ({
...
+        newVersion={newVersion}
         onUpdateClicked={onUpdate}
       />
     ),
     onClose: onDismiss,
   });
 }
+export function onAutoupdateInstalled({ newVersion }: { newVersion: string }) {
+  openToast('compass-update-restarted', {
+    variant: 'note',
```
Comment (gribnoysup) [inline packages/compass/src/app/components/update-toasts.tsx:138]: This seems to be success in the designs, not note

Comment (mabaasit) [reply packages/compass/src/app/components/update-toasts.tsx]: updated in [a19f91d](https://github.com/mongodb-js/compass/pull/5718/commits/a19f91d28aec1aa5d06f6f394ffeb955f6410b73)

File: packages/compass-e2e-tests/helpers/commands/hide-toasts.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function hideToasts(browser: CompassBrowser): Promise<void> {
```
Comment (gribnoysup) [inline packages/compass-e2e-tests/helpers/commands/hide-toasts.ts:4]: Super nit on the naming

```suggestion
export async function hideAllVisibleToasts(browser: CompassBrowser): Promise<void> {
```

**Review by gribnoysup — APPROVED** [review]

File: packages/compass/src/app/components/update-toasts.tsx (hunk 1 above)
Comment (mcasimir) [reply packages/compass/src/app/components/update-toasts.tsx:16]: TIL!

**Pull request merged**